mkdocs_monorepo_plugin/helpers.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 Spotify AB
```

Comment:
I love how flat / low-complexity all of these functions are!  Makes it a little easier to understand what's going on.